plugins/services/src/js/structs/Pod.d.ts

```diff
@@ -12,4 +12,5 @@ export default class Pod extends Service {
   getVolumesData(): any;
   getRegions(): any;
   isDelayed(): boolean;
+  getRole(): string;
```

Comment:
Since this `extends Service` we shouldn't need to add `getRole` to both places right? Pod should inherit `getRole` from `Service`.